docs/source/support/release-notes.rst

```diff
@@ -9,6 +9,15 @@ This page contains release notes for the SDK.
 HEAD — ongoing
 --------------
 
+Ledger
+~~~~~~
+
+* Do not compile packages twice when loading a scenario with ``--scenario``, see
+  `issue #1238 <https://github.com/digital-asset/daml/issues/1238>`__.
+* Allow to load packages eagerly when starting the sandbox, which is useful
```

Comment:
Similar to the other comment. I guess this should be something like "When starting the sandbox, you can now load packages 'eagerly'" - maybe explain what that means? "which is useful ..." and as before.

Reply:
Reworded, please check